### ISSUE TITLE
How the extension used?

### ISSUE THREAD

**Issue by kentr** [rupisaini/vscode-ansible-linter#7]
I've installed the `vscode-ansible-linter` extension, and it doesn't appear to do anything on Ansible task files.  There are also no commands appearing for the extension.

I have `ansible-lint` installed and can run it via command line.

Environment:
* VS Code `1.31.1`
* MacOS High Sierra
* ansible-lint `3.4.2`

**Comment by rupisaini** [comment]
When you run ansible-lint from command line is it returning you any error?

**Comment by kentr** [comment]
@rupisaini 

I'm not very familiar with `ansible-lint`, but AFAIK the command works as-expected.   It lints the project and reports linting errors.

**Comment by tristanbes** [comment]
true, nothing happens. I can use `ansible-lint` on the terminal, but nothing is shown when I open .yml files on VSCode. I've used your extension in the past, it was great but it simply doesn't work now :(

**Comment by rupisaini** [comment]
Hi,

I am not writing ansible recently, but I will definitely look into this
over the weekend. Can you please share environment details i.e. VSC
version, os etc?

**Comment by kentr** [comment]
@rupisaini 

* ansible-lint `3.4.23`
* Visual Studio Code `1.31.1`
* MacOS `10.13.6` (High Sierra)

**Comment by dacrhu** [comment]
Working in command line:

[dacr@dacr]$ ansible-lint playbooks/0.yml 
[201] Trailing whitespace
/home/dacr/develop/........../tasks/application_server.yml:24

Visual Studio Code 1.31.0
Fedora 29
ansible-lint 4.1.0

But in VSC no menu, no debug, nothing.

**Comment by mark1709** [comment]
+1

**Comment by kttyt** [comment]
+1

**Comment by FlavioAmurrioCS** [comment]
+1

**Comment by thinkedem** [comment]
+10

**Comment by EpiqSty** [comment]
+100

**Comment by ssbarnea** [comment]
Try https://marketplace.visualstudio.com/items?itemName=zbr.vscode-ansible -- that one should work. I am the author of that newer extension and also the current maintainer of ansible-lint.
